core/amber/src/main/python/core/models/tuple.py

```diff
@@ -0,0 +1,20 @@
+from abc import ABC
+from dataclasses import dataclass
+
+import pandas
+
+
+@dataclass
+class Tuple(ABC):
+    """
+    Python representation of the Texera.Tuple
+    """
+
+
+# Use pandas.Series as a Tuple.
+Tuple.register(pandas.Series)
```

Comment:
Can you be more specific about this comment because this python function of `register` might not be very familiar to others. Does this mean `Tuple` and `pandas.Series` can be used interchangeably?

Reply:
done